gnosis/gp-ui
# 232/filled row logic

# Summary

Closes #232 

Building up on the UI added by Michel, this PR wires up the logic

# Testing

- Check any order against the PR build
- Check the storybook
